app/views/application/_new.html.erb

```diff
@@ -0,0 +1,24 @@
+<div>
+  <h2>New <%= media.class %></h2>
+  <ul>
+  <% media.errors.each do |column, message| %>
+    <li><%=column.capitalize %> <%= message %></li>
+  <% end %>
+</ul>
+
+  <%= form_for media do |f| %>
+    <div class="form-group">
+      <%=f.label :name %>
+      <%=f.text_field :name, class: "form-control" %>
+    </div>
+    <div class="form-group">
+      <%= f.label maker %>
+      <%= f.text_field maker, class: "form-control" %>
+    </div>
+    <div class="form-group">
+      <%= f.label :description %>
+      <%= f.text_area :description, class: "form-control"  %>
+    </div>
+    <%= f.submit "Save", :class => "btn btn-default" %>
+  <% end %>
+</div>
```

Comment:
Is this form identical to _edit.html.erb except for the `New` and `Edit` on line 2? You could probably DRY this up by having one form.